README.md

```diff
@@ -824,13 +824,18 @@ You can add a registration button to the primary auth page by setting the follow
 ```javascript
 // An example that adds a registration button underneath the login form on the primary auth page
 registration: {
-  click: function(){
+  enable: true,
+  click: function() {
     window.location.href = 'https://acme.com/sign-up';
+  },
+  submit: function(registrationData) {
+    // submit the registration data
   }
 }
 ```
+- **registration.enable** - Display the registration section in the primary auth page. Defaults to `false`.
 - **registration.click** - Function that is called when the registration button is clicked
-
+- **registration.submit** - Function that is called when the registration form is submitted.
```

Comment:
I think we should consider doing this the way that we talked about a couple days ago - i.e. if we're changing the signature, it's better to do it once (because it will be a breaking change, or at least unclear, in the very next update).

Specifically, there should be two hooks here:

1. updateSchema (or something like that) - step that happens after the schema request is made and they get a chance to change it. In our case, since no endpoint exists yet, we should just pass in our default schema that you defined elsewhere into this function. Goal here is that when we move to the endpoint, their code doesn't need to change, and our code just needs to swap out our hardcoded schema with the result of the request. **Note** this should support async - they should have the option to do async logic and then continue on.

2. submit - hmmm, so feels like there should be a couple behaviors here. Maybe we need another method. What we want is this:

- They can override completely, and just get results of form
- They can return schema + values, which is then passed to our registration endpoint. Should support async as well.

Looking at it now (and I see where you're going with this) - ideally we can do just the submit part and not have it conflict with later changes (i.e. maybe we don't need to do the updateSchema hook yet). If so, I'm okay with that - but what is the plan for how submit behaves (specifically when we're going to pass it through to the RAAS endpoint)?